### PR TITLE
Add the missing exports condition to support vite-plugin-svelte v3

### LINKED ISSUE #177: Fix warning while using with `vite-plugin-svelte`
While using the dep with `@sveltejs/kit`:`v2.x` setup, vite:`v5.x` and `@sveltejs/vite-plugin-svelte`:`v3.0.0` on running the app a warning is displayed:

```
Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.
[vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-portal@2.2.0
```

### PR DESCRIPTION
**Pull request by harshmandan**
Fixes #177

### CODE REVIEW

**Comment by ebeloded** [comment]
Can this be merged please, @romkor?

File: package.json [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  "version": "2.2.1",
   "description": "Svelte component for rendering outside the DOM of parent component",
   "main": "src/main.cjs.js",
   "module": "src/main.es.js",
   "svelte": "src/Portal.svelte",
   "types": "src/Portal.svelte.d.ts",
+  "exports": {
+    ".": {
+      "types": "./src/Portal.svelte.d.ts",
+      "svelte": "src/main.es.js"
```
Comment (romkor) [inline package.json:12]:
```suggestion
    ".": {
      "types": "./src/Portal.svelte.d.ts",
      "svelte": "src/main.es.js"
    }
```
Please add closing brace

File: package.json [hunk 2]
```diff
@@ -1,11 +1,16 @@
 {
   "name": "svelte-portal",
-  "version": "2.2.0",
+  "version": "2.2.1",
```
Comment (romkor) [inline package.json:3]:
```suggestion
  "version": "2.2.0",
```
Version will be bumped automatically

**Review by romkor — CHANGES_REQUESTED** [review]

**Comment by romkor** [comment]
@harshmandan just released

**Pull request merged**